settings.gradle

```diff
@@ -9,6 +9,11 @@ include "spock-junit4"
 include "spock-temp"
 include "spock-testkit"
 
+if ((System.getProperty("variant") as BigDecimal ?: 3.0) != 3.0) {
```

Comment:
You now have inconsistent default behavior.
In `build.gradle` you default to 2.5, here you default to 3.0.
So if you just run without the system property, the compat project is missing.

Reply:
Nope, I have 3.0 by default everywhere:
![image](https://user-images.githubusercontent.com/148013/72425787-b6318500-3788-11ea-99d2-e90341f234ab.png)

Reply:
Ah, sorry, then I misinterpreted `please change the order to 2.5, 3.0` from @leonard84, I thought it also meant to use `2.5` as default, not only the visual order.

Reply:
I was thinking about that, but for me Groovy 3 is the future and builds from `master` should stay with the newest version (when no specified). Especially that there is a change of the major version number in Spock.

Reply:
Yes I meant that `2.5` should be the default if no `-Dvariant=3.0` is added, we always had the lowest supported version as default, as that was the most likely to break if new features were used

Reply:
I like to be cutting edge and see less Java 9+ warnings :-/. Nevertheless, if you like to detect more issues in IDE I changed to 2.5 by default.